ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -56,6 +56,11 @@ tests:
   container:
     from: src
   run_if_changed: \.go$
+- as: verify-generate
+  commands: make verify-generate VERSION=5.0.2-dev
```

Comment:
Shouldn't we be at 5.0.3 now?

Reply:
The current one is generated with `5.0.2-dev`:

https://github.com/openshift/cincinnati-operator/blob/95257fc4670dc209befcc9c93f896ddba4371efd/bundle/manifests/update-service-operator.clusterserviceversion.yaml#L282

I can bump it to `5.0.3-dev` with another PR if desired.

Reply:
Hmm I'm not too happy about carrying this version in o/release CI job config. Maybe we should carry this version somewhere in the operator repo (in Makefile?) and "official" bumps would change it there?

Reply:
`Version` in `Makefile` sounds good to me.
It is there already. I will modify its value.

https://github.com/openshift/cincinnati-operator/blob/2bbe7bf15c1f936fdd672037777b0c040a982dda/Makefile#L1-L2

Reply:
https://github.com/openshift/cincinnati-operator/pull/205